scripts/combine/pullsAndImpacts.py

```diff
@@ -189,8 +258,8 @@ def readFitInfoFromFile(rf,filename, group=False, sort=None, ascending=True, sta
     constraints = np.zeros_like(labels, dtype=float)
     if not group:
         import ROOT
-        rtfile = ROOT.TFile.Open(filename)
-        rtree = rtfile.Get(treename)
+        fitresult = ROOT.TFile.Open(filename.replace(".hdf5",".root"))
```

Comment:
Does it read from the hdf5 file or only root?

Reply:
the ttree is currently only stored in root, there is no issue with the object getting too large. It seems a bit more effort in the combinetf script to store everything, including the ttree, in hdf5 and is essentially not necessary. So the script currently uses both fitresult files, the hdf5 and root file.